packages/puppeteer-core/src/cdp/CdpSession.ts

```diff
@@ -74,6 +74,10 @@ export class CdpCDPSession extends CDPSession {
     return this.#connection;
   }
 
+  override get detached(): boolean {
```

Comment:
Nit: you can remove the private  `#closed` getter as getter value can't be modified, so no need to have two thing that are the same.

Reply:
Done, thanks for the suggestion!